inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/evaluation/EvaluationResult.java

```diff
@@ -36,7 +36,13 @@
 {
     private final int trainingSetSize;
     private final int testSetSize;
+    
+    /**
+     * Rate of this training data compared to all training data
+     */
+    private final double trainingDataRatio;
     private boolean skippedEvaluation;
+    private String errorMsg;
```

Comment:
I would like that String to be Optional

Reply:
I think we should avoid `Optional` fields. If it makes sense to make the getter optional, then return `Optional.ofNullable(...)`.